### PR TITLE
Mechanical rename of everything

### PR DESCRIPTION
**Pull request by turian**

### CODE REVIEW

File: torchsynth/module.py [hunk 1]
```diff
@@ -150,7 +62,7 @@ def seconds_to_samples(self, seconds: T) -> T:
 
     def _forward(self, *args: Any, **kwargs: Any) -> Signal:  # pragma: no cover
         """
-        Each TorchSynthModule should override this.
+        Each TorchSynthModule0Ddeprecated should override this.
```
Comment (jorshi) [inline torchsynth/module.py:65]: why is this TorchSynthModule0Ddeprecated?

File: torchsynth/module.py [hunk 2]
```diff
@@ -600,14 +512,14 @@ def _forward(self, control_in: Signal, audio_in: Signal) -> Signal:
...
-class TorchNoise(TorchSynthModule1D):
+class TorchNoise(TorchSynthModule):
     """
     Adds noise to a signal
 
     Parameters
     ----------
     ratio (float): mix ratio between the incoming signal and the produced noise
-    **kwargs: see TorchSynthModule
+    **kwargs: see TorchSynthModule0Ddeprecated
```
Comment (jorshi) [inline torchsynth/module.py:522]: Same as above

Comment (turian) [reply torchsynth/module.py]: mechanical error. I fixed in later PR but push here anyway.

File: torchsynth/module.py (hunk 1 above)
Comment (turian) [reply torchsynth/module.py]: mechanical error. I fixed in later PR but push here anyway.

**Pull request merged**